app/repositories/post.py

```
@@ -0,0 +1,272 @@
+from app.utils.id_generator import generate_post_id
+
+
+class PostRepository(BaseRepository):
+    """
+    게시글 데이터 저장소
+    
+    Attributes:
+        data_dir: 데이터 디렉토리 경로
+    """
+    
+    def __init__(self, data_dir: str = "./data"):
```

Ruff (0.14.11)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)